src/dcp/dcp.c

```diff
@@ -135,6 +140,16 @@ int main(int argc, char** argv)
         }
 
         switch(c) {
+            case 'b':
+                if (mfu_abtoull(optarg, &megs) != MFU_SUCCESS || megs == 0) {
+                    if (rank == 0) {
+                        MFU_LOG(MFU_LOG_ERR,
+				"Failed to parse block size: '%s'\n", optarg);
+                        usage = 1;
+                    }
+                }
+                mfu_copy_opts->block_size = (size_t)megs * 1024 * 1024;
```

Comment:
@adilger , the mfu_abtoull function parses strings like "1MB" already, so this line would actually change such a string to produce a value of 1GB.  Let's just have the user specify the size in bytes rather than MB.  They'll have to remember to give an M or MB suffix on the value, but I think our hint of a 1MB default should be a good reminder for them.

Reply:
Since the default transfer size is 1 MB, and if the user provides an input size that is too small would just result in an error (which IMHO isn't very useful), I think it would make sense to accept values that are small (say < 1024) as units of MB, and if any units are given the values will always be large enough to avoid this case.